packages/eas-cli/src/build/local.ts

```diff
@@ -8,8 +8,14 @@ import { ora } from '../ora';
 const PLUGIN_PACKAGE_NAME = 'eas-cli-local-build-plugin';
 const PLUGIN_PACKAGE_VERSION = '0.0.120';
 
+export enum LocalBuildMode {
+  DISABLED,
+  LOCAL_BUILD_PLUGIN,
+  INTERNAL,
+}
```

Comment:
I'm not sure what this is and how it's supposed to work.

Reply:
Add a comment?

Reply:
I agree it would be good to have some description of what each of these means and does